R/pkg/R/mllib_clustering.R

```diff
@@ -404,11 +411,14 @@ setMethod("summary", signature(object = "LDAModel"),
             vocabSize <- callJMethod(jobj, "vocabSize")
             topics <- dataFrame(callJMethod(jobj, "topics", maxTermsPerTopic))
             vocabulary <- callJMethod(jobj, "vocabulary")
+            trainingLogLikelihood <- callJMethod(jobj, "trainingLogLikelihood")
+            logPrior <- callJMethod(jobj, "logPrior")
```

Comment:
I think it's more appropriate to return ```NULL``` rather than ```NaN``` for local LDA model, since the ```logPrior``` is not existing rather than not a number.
BTW, I think we can return NULL directly according to ```isDistributed```, otherwise, call corresponding Scala methods. This should reduce the complexity of ```LDAWrapper``` and reduce communication between R and Scala.